x-pack/plugins/watcher/public/application/sections/watch_edit_page/components/monitoring_watch_edit/monitoring_watch_edit.tsx

```diff
@@ -28,20 +28,18 @@ export const MonitoringWatchEdit = ({ pageTitle }: { pageTitle: string }) => {
   );
 
   return (
-    <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
-      <EuiEmptyPrompt
-        iconType=""
-        title={<h1>{pageTitle}</h1>}
-        body={<p>{systemWatchMessage}</p>}
-        actions={[
-          <EuiLink {...reactRouterNavigate(history, `/watches/watch/${watch.id}/status`)}>
-            <FormattedMessage
-              id="xpack.watcher.sections.watchEdit.monitoring.header.watchLinkTitle"
-              defaultMessage="View watch status"
-            />
-          </EuiLink>,
-        ]}
-      />
-    </EuiPageContent>
+    <EuiPageTemplate.EmptyPrompt
```

Comment:
didn't verify this visually since I'm not sure, monitoring watches are still in use. The code changes LGTM